packages/plugin-assets/src/commands/assets/init.js

```diff
@@ -1,19 +1,23 @@
 const { flags } = require('@oclif/command');
 const { TwilioClientCommand } = require('@twilio/cli-core').baseCommands;
 const { getPluginConfig } = require('../../pluginConfig');
+const generateProjectName = require('project-name-generator');
```

Comment:
We recently had some issues with name generators generating offensive names. Do we feel comfortable with this one?

Reply:
It appears this would have been a problem, but someone already pruned the adjectives and nouns for problematic words/potential combinations in [this pull request](https://github.com/aceakash/project-name-generator/pull/33) and previously in [this pull request](https://github.com/aceakash/project-name-generator/pull/21).